main.go

```diff
@@ -1,14 +1,53 @@
 package main
 
 import (
+	_ "embed"
+	"fmt"
+
 	fzf "github.com/junegunn/fzf/src"
 	"github.com/junegunn/fzf/src/protector"
 )
 
 var version string = "0.47"
 var revision string = "devel"
 
+//go:embed shell/key-bindings.bash
+var bashKeyBindings []byte
+
+//go:embed shell/completion.bash
+var bashCompletion []byte
+
+//go:embed shell/key-bindings.zsh
+var zshKeyBindings []byte
+
+//go:embed shell/completion.zsh
+var zshCompletion []byte
+
+//go:embed shell/key-bindings.fish
+var fishKeyBindings []byte
+
 func main() {
 	protector.Protect()
-	fzf.Run(fzf.ParseOptions(), version, revision)
+	options := fzf.ParseOptions()
+	if options.Bash {
+		fmt.Println("### key-bindings.bash ###")
+		fmt.Println(string(bashKeyBindings))
+		fmt.Println("### completion.bash ###")
+		fmt.Println(string(bashCompletion))
```

Comment:
I think the suggested `'/### completion/,$p'` and `'/### completion/,$d'` are not forward compatible. These sed formulae can be broken when the order of the section is swapped or when another new section is added. I think it would be more robust to have terminators like

```suggestion
		fmt.Println("### end ###")
		fmt.Println("### completion.bash ###")
		fmt.Println(string(bashCompletion))
		fmt.Println("### end ###")
```

so that users can specify
- `sed -n '/^### completion/,/^### end/p'`
- or `sed -n '/^### key-bindings/,/^### end/p'`
- or `sed -n '/^### key-bindings/,/^### end/p;/^### completion/,/^### end/p'`.

Reply:
Good point!

Reply:
2b26315cf4342c3578fe13da52ea93dbe0b61212